src/test/java/jewellery/inventory/unit/service/ResourceInUserServiceTest.java

```diff
@@ -38,6 +40,8 @@
 @ExtendWith(MockitoExtension.class)
 class ResourceInUserServiceTest {
   @Mock private UserRepository userRepository;
```

Comment:
From what I saw UserRepository and ResourceRepository are no longer used in ResourceInUserService. Please, remove them from the mocks in the tests.